examples/s4u/actor/s4u_actor.cpp

```diff
@@ -0,0 +1,87 @@
+/* Copyright (c) 2006-2015. The SimGrid Team. All rights reserved.          */
```

Comment:
The name of this test should probably be changed to include "mutex" inside it if the tests is supposed to be about mutexes.